hypothesis-python/src/hypothesis/extra/django/_fields.py

```diff
@@ -231,6 +231,22 @@ def register_field_strategy(field_type, strategy):
     _global_field_lookup[field_type] = strategy
 
 
+def get_string_field_bounds(field):
```

Comment:
I think you intended this to be run for the text function above, rather than the generic from_field?